setup.py

```diff
@@ -17,28 +17,33 @@
 URL                 = 'http://scikit-image.org'
 LICENSE             = 'Modified BSD'
 DOWNLOAD_URL        = 'http://github.com/scikit-image/scikit-image'
-VERSION             = '0.12dev'
-PYTHON_VERSION      = (2, 6)
+SETUP_REQUIRES = ['cython>=0.21']
+INSTALL_REQUIRES = """
+cython>=0.21
+matplotlib>=1.1.0
+numpy>=1.6.1
+scipy>=0.9
+six>=1.3
+networkx>=1.8
+pillow>=1.7.8
+""".strip().splitlines()
 
+import setuptools
 import os
 import sys
 
-import setuptools
 from distutils.command.build_py import build_py
-from distutils.version import LooseVersion
 
 
-# These are manually checked.
-# These packages are sometimes installed outside of the setuptools scope
-DEPENDENCIES = {}
-with open('requirements.txt', 'rb') as fid:
-    data = fid.read().decode('utf-8', 'replace')
-for line in data.splitlines():
-    pkg, _, version_info = line.replace('==', '>=').partition('>=')
-    # Only require Cython if we have a developer checkout
-    if pkg.lower() == 'cython' and not VERSION.endswith('dev'):
-        continue
-    DEPENDENCIES[str(pkg).lower()] = str(version_info)
+with open('skimage/__init__.py') as fid:
+    for line in fid:
+        if line.startswith('__version__'):
+            VERSION = line.strip().split()[-1][1:-1]
```

Comment:
Its just setting a global variable based on a value in another file.  Matplotlob does the same thing.